perses/rjmc/geometry.py

```diff
@@ -474,30 +485,8 @@ def _angle_logp(self, theta, angle, beta):
         """
         k_eq = angle.type.k*units.kilocalories_per_mole/(units.radians**2)
         theta0 = angle.type.theteq*units.radians
```

Comment:
I think `angle.type.theteq` is actually in degrees.

@swails uses `DEG_TO_RAD` to convert these to radians when he uses these values:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L1905